backend/de.metas.handlingunits.base/src/main/java/de/metas/adempiere/gui/search/impl/OLCandHUPackingAware.java

```diff
@@ -75,14 +75,14 @@ public void setM_Product_ID(final int productId)
 	@Override
 	public void setQty(final BigDecimal qty)
 	{
-		olCand.setQtyEntered(qty);
+		olCand.setQtyEntered_Override(qty);
```

Comment:
why this?

Reply:
because otherwise we risk to not set the actual qty
it is the same for all fields that can be overridden (e.g. `M_Product`)